test_absolute (__main__.TestExercice) ... FAIL
test_factorial (__main__.TestExercice) ... FAIL
test_prefixes (__main__.TestExercice) ... FAIL
test_summation (__main__.TestExercice) ... ERROR
test_verify_ages (__main__.TestExercice) ... FAIL

======================================================================
ERROR: test_summation (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 45, in test_summation
    output = exercice.prime_integer_summation()
  File "/home/runner/work/2021h-ch5-exercices-Ismail733/2021h-ch5-exercices-Ismail733/exercice.py", line 29, in prime_integer_summation
    for z in range (2,v,1):
TypeError: 'float' object cannot be interpreted as an integer

======================================================================
FAIL: test_absolute (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 22, in test_absolute
    'Mauvaise réponse'
AssertionError: Lists differ: [0, 0] != [5, 5]

First differing element 0:
0
5

- [0, 0]
+ [5, 5] : Mauvaise réponse

======================================================================
FAIL: test_factorial (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 62, in test_factorial
    'Mauvaise réponse'
AssertionError: Lists differ: [0, 0, 0, 0] != [1, 6, 40320, 3628800]

First differing element 0:
0
1

- [0, 0, 0, 0]
+ [1, 6, 40320, 3628800] : Mauvaise réponse

======================================================================
FAIL: test_prefixes (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 32, in test_prefixes
    'Mauvaise réponse'
AssertionError: Lists differ: [None] != ['Jack', 'Kack', 'Lack', 'Mack', 'Nack', 'Oack', 'Pack', 'Qack']

First differing element 0:
None
'Jack'

Second list contains 7 additional elements.
First extra element 1:
'Kack'

- [None]
+ ['Jack', 'Kack', 'Lack', 'Mack', 'Nack', 'Oack', 'Pack', 'Qack'] : Mauvaise réponse

======================================================================
FAIL: test_verify_ages (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 77, in test_verify_ages
    'Mauvaise réponse'
AssertionError: Lists differ: [] != [False, True, False, False, True, True, True, True, False]

Second list contains 9 additional elements.
First extra element 0:
False

- []
+ [False, True, False, False, True, True, True, True, False] : Mauvaise réponse

----------------------------------------------------------------------
Ran 5 tests in 0.001s

FAILED (failures=4, errors=1)
